.gitignore

```diff
@@ -19,3 +19,6 @@
 
 # Go workspace file
 go.work
+
+# IntelliJ files
+.idea
```

Comment:
This should be on your personal gitignore. `git config --global core.excludesfile ~/.gitignore_global`

Reply:
Done ✅